src/native/eventpipe/ep.h

```diff
@@ -8,6 +8,22 @@
 #include "ep-stack-contents.h"
 #include "ep-rt.h"
 
+#if FEATURE_CORECLR
```

Comment:
better to but this into each runtimes ep-rt.h header, that's where we keep runtime specializations.